src/main/java/randoop/generation/MemoryClassLoader.java

```diff
@@ -0,0 +1,46 @@
+package randoop.generation;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/** A class loader that loads classes from in-memory data. */
```

Comment:
This documentation does not say anything about instrumentation, but that is what the class really does.  The class should perhaps be renamed as well.